ChIP-Seq/tutorials/TAL1_binding_site_identification.md

```diff
@@ -0,0 +1,420 @@
+
+    <img src="../images/Highlighting_viz_button.png"/>   
+    <figcaption><b>Figure X:</b> Trackster can be accessed from the Visualizations button at the top of the screen.</figcaption>
+    
+
+
+4. Give this session a descriptive name and choose 'mm10' as the 'Reference genome build (dbkey)'.  
+    
+    <img src="../images/Trackster_session_and_db.png"/>   
+    <figcaption><b>Figure X:</b> Session name and assigning reference genome.</figcaption>
+
+
+
+5. Click 'Add Datasets to visualization' and select the history containing the data from this analysis. Select the bedgraph files and the peak files that you renamed.
+
+    <img src="../images/Trackster_add_datasets.png"/>   
+    <figcaption><b>Figure X:</b> Load your data into Trackster with the 'Add Datasets to visualization' feature. </figcaption>
+
+
+     <img src="../images/Trackster_selecting_datasets.png"/>   
+    <figcaption><b>Figure X:</b> Select data from your histories to view in Trackster. </figcaption>
+
+6. Navigate to the Runx1 locus (chr16:92501466-92926074) to inspect the aligned reads and Tal1 peak calls. 
+
+    | :grey_question: Questions |
+    |:---|
+    | <ul><li>What do you see?</li><li>What gene(s) other than Runx1 could be regulated by Tal1?</li></ul> |
+    
+
+
+    <img src="../images/Trackster_Runx1_locus.png"/>   
+    <figcaption><b>Figure X:</b> The Runx1 locus.</figcaption>
+
+<a name="step9"/></a>
+### Step 9: Identifying unique and common Tal1 peaks between the G1E and megakaryocyte states
+
+We've just processed ChIP-seq data from two stages of hematopoiesis and have lists of Tal1 occupied sites (peaks) in both cellular states. Now let's identify Tal1 peaks that are shared between the two cellular states and also those that are specific to one cellular state.
+
+:pencil2: ***Hands on!***
+
+1. Run the tool `bedIntersect` to find peaks that exist both in G1E and megakaryocytes. 
+
+  - Select the "Tal1 G1E peaks" and "Tal1 Mega peaks" files as the inputs. Running this tool with the default settings will return overlapping peaks of both files. 
+  
+2. Run the tool `bedIntersect` to find peaks that exist only in G1E.
+
+  - Select "Tal1 G1E peaks" as the first input and "Tal1 Mega peaks" as the second input file.
+  - **Report only those alignments that \*\*do not\*\* overlap the BED file**: Yes
+  
+3. Run the tool `bedIntersect` to find peaks that exist only in megakaryocytes.
+
+  - Select "Tal1 Mega peaks" as the first input and "Tal1 G1E peaks" as the second input file.
+  - **Report only those alignments that \*\*do not\*\* overlap the BED file**: Yes  
+  
+4. Re-name the three files we generated to reflect their contents. 
+
+    | :grey_question: Questions |
+    |:---|
+    | <ul><li>How many Tal1 binding sites are common to both G1E cells and megakaryocytes?</li><li>How many are unique to G1E cells?</li><li>How many are unique to megakaryocytes?</li></ul> |
+
+<a name="stepA"/></a>
+# OPTIONAL ADDITIONAL ANALYSES
```

Comment:
I have reorganized the main Steps with some advice from @friedue, and have moved the GC bias section to the Optional Additional Analyses section.